handlers/spaces.py

```
@@ -70,7 +70,7 @@ async def check_transaction_status(signature: str, command_start_time: datetime,
             signature_obj = Signature.from_string(signature)
         except Exception as e:
             logger.error(f"Error converting signature format: {str(e)}")
-            return False, f"❌ Error converting signature format: {str(e)}"
+            raise TransactionError("Error converting signature format", "SIGNATURE_CONVERSION_ERROR")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)